File: Day-09/Pattern10.py
n = int(input())

def PatternTen(n):
    i = 1
    while(i <=n):
        count = i
        j = 1
        while(j <=i):
            print(count, end='')
            j = j+1
            count = count -1
        print(end='\n')
        i = i+ 1
    

PatternTen(n)


def PatternTenNoCount(n):
    i = 1
    while(i <=n):
        j = 1
        while(j <=i):
            print(i-j+1, end='')
            j = j+1
        print(end='\n')
        i = i+ 1

PatternTenNoCount(n)

File: Day-09/Pattern10.js
function PatternTen(n) {
  let i = 1;
  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= i) {
      str += i - j + 1;
      j++;
    }
    console.log(str);
    i++;
  }
}

PatternTen(3);
